README.md

```diff
@@ -1 +1,60 @@
-# Hello-Micoverse-Project
\ No newline at end of file
+
+# Hello Microverse
+
+> This is an introductory project by microverse to test creating of repository, branches, linters and Pull Request, 
+
+![screenshot](./screenshot.png)
+
+
+## Built With
+
+- Html
+- CSS
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Github Account 
+
+### Setup
+Set up and configure git on your system.
+
+### Install
+Clone this repository
+
+
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@ifnotlily](https://github.com/ifnotlily)
+- Twitter: [@ifnotlily](https://twitter.com/ifnotlily)
+- LinkedIn: [lilian-francis](https://linkedin.com/in/lilian-francis)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- Kindly add more details in how to install and follow steps for non-technical people in your project.